### PR TITLE
Updated dependencies, cleaned up

### PR DESCRIPTION
**Pull request by VEINHORN**

### CODE REVIEW

File: project/Settings.scala [hunk 1]
```diff
@@ -5,7 +5,7 @@ import sbtassembly.AssemblyKeys._
 object Settings {
   val buildName = "hazelcast-spark"
   val buildVersion = "0.1-SNAPSHOT"
-  val buildScalaVersion = "2.10.5"
+  val buildScalaVersion = "2.11.8"
```
Comment (eminn) [inline project/Settings.scala]: From Spark documentation : Spark 1.6.2 uses Scala 2.10. You will need to use a compatible Scala version (2.10.x).

**Comment by eminn** [comment]
Thanks for your contribution. Could you please sign the Hazelcast Contributor Agreement ? https://hazelcast.atlassian.net/wiki/display/COM/Hazelcast+Contributor+Agreement

**Comment by VEINHORN** [comment]
@eminn Hi! I signed it and sent to info@hazelcast.com

**Comment by VEINHORN** [comment]
@eminn Hi! I signed it and sent to info@hazelcast.com

File: project/Settings.scala (hunk 1 above)
Comment (VEINHORN) [inline project/Settings.scala]: @eminn maybe there is some sense to update Scala to 2.10.6 It's last version in 2.10.*

Comment (eminn) [inline project/Settings.scala]: @VEINHORN yes, it makes sense to update 2.10.6

Comment (VEINHORN) [inline project/Settings.scala]: @eminn I moved to Scala 2.10.6 but I don't know how to clearly remove this [commit](https://github.com/VEINHORN/hazelcast-spark/commit/9ee17ede2875e0d77132490d08ea96c52a5b458e) Maybe you have some ideas about it.

Comment (eminn) [inline project/Settings.scala]: You can rewrite your commit history with interactive rebase in git using `git rebase -i master` on your branch. You can have a look at git rebase tutorials on the web.

**Pull request merged**